releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,12 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v7.0.0 - November, 20 2023
+**Breaking changes**
+  - Updated to fail the task if any Key Vault references cannot be resolved or if the resolved values are invalid to be set as the pipeline secret.
+
+* Added an option to treat Key Vault resolution errors as warnings.
```

Comment:
It doesn't seem like the new option we've added is a breaking change. In that case it should be under a different header than "Breaking changes"

Reply:
I have added another header **Enhancement** let me know if thats okay